website/pages/guides/[[...page]].jsx

```diff
@@ -1,33 +1,37 @@
 import { productName, productSlug } from 'data/metadata'
-import order from 'data/guides-navigation.js'
 import DocsPage from '@hashicorp/react-docs-page'
+// Imports below are only used server-side
 import {
   generateStaticPaths,
   generateStaticProps,
 } from '@hashicorp/react-docs-page/server'
 
-const subpath = 'guides'
+//  Configure the docs path
+const BASE_ROUTE = 'guides'
+const NAV_DATA = 'data/guides-nav-data.json'
+const CONTENT_DIR = 'content/guides'
+// override default "main" value for branch for "edit on this page"
+const MAIN_BRANCH = 'master'
 
-export default function GuidesLayout(props) {
+function DocsLayout(props) {
```

Comment:
The `guides` and `intro` routes provided an opportunity to prove out the "core use case" (local content only) for the [MKTG-032](https://docs.google.com/document/d/1kYvbyd6njHFSscoE1dtDNHQ3U8IzaMdcjOS0jg87rHg) revisions we've made to DocsPage and DocsSidenav components.

Pages under these routes will render identically to how they do now, the only difference is they now use the updated `nav-data.json` format.